lib/reek/cli/input.rb

```diff
@@ -1,7 +1,7 @@
 require 'reek/source'
 
 module Reek
-  module CLI
+  module Cli
```

Comment:
It's `Cli` everywhere else in the codebase. I can change it all to `CLI` if you prefer. I agree that would make more sense.